models/enterprise_sonic/logging/merged_example_01.txt

```diff
@@ -0,0 +1,34 @@
+# Using merged
+#
+# Before state:
+# -------------
+#
+sonic# show logging servers
+--------------------------------------------------------------------------------
+HOST            PORT      SOURCE-INTERFACE    VRF            MESSGE-TYPE
+--------------------------------------------------------------------------------
+10.11.1.1       616       Ethernet8           -              log
+
+- name: Merge logging server configuration
+  sonic_logging:
+    config:
+      remote_servers:
+        - host: 10.11.1.2
+          remote_port: 5
+          source_interface: Ethernet24
+          message_type: event
+        - host: log1.dell.com
+          remote_port: 6
+          source_interface: Ethernet28
+    state: merged
+
+# After state:
+# ------------
+#
+sonic# show logging servers
+--------------------------------------------------------------------------------
+HOST            PORT      SOURCE-INTERFACE    VRF            MESSGE-TYPE
+--------------------------------------------------------------------------------
+10.11.0.2       5         Ethernet24          -              event
+10.11.1.1       616       Ethernet8           -              log
+log1.dell.com   514       Ethernet28          -              log
```

Comment:
Why does this "show" output indicate "PORT: 514" instead of "PORT: 6" as specified in the merged command?

Reply:
Changed to 6